src/field.rs

```diff
@@ -443,6 +455,7 @@ mod tests {
         assert_matches!(result, Err(FieldError::InputSizeMismatch));
     }
 
+    #[allow(clippy::eq_op)]
```

Comment:
What's the clippy warning that's beeing suppressed here?

Reply:
[eq_op](https://rust-lang.github.io/rust-clippy/master/index.html#eq_op): clippy warns about seemingly tautological comparisons, like `assert_eq!(one - one, zero);` and `assert_eq!(two / two, one);` in this function. In our case, it makes sense to check that such obvious seeming expressions evaluate as expected.

Unfortunately you can't put the `#[allow]` directly on the offending lines because ["`attributes on expressions are experimental`"](https://github.com/rust-lang/rust/issues/15701). I'll add a comment explaining what we're silencing.

Reply:
looks good.